tests/assert-jobs/utils/params.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"github.com/spf13/viper"
+	"time"
+)
+
+const (
+	jaegerInstanceNameKey = "JAEGER_INSTANCE_NAME"
```

Comment:
If you keep this test generic, agnostic from Kubernetes, you could use this to test a local Jaeger as well.